app/src/main/java/woowacourse/omok/controller/InitController.kt

```diff
@@ -0,0 +1,47 @@
+package woowacourse.omok.controller
+
+import android.widget.ImageView
+import android.widget.TableLayout
+import android.widget.TableRow
+import androidx.core.view.children
+import omok.domain.BlackStone
+import omok.domain.Board
+import omok.domain.WhiteStone
+import omok.domain.state.Turn
+import woowacourse.omok.MainActivity
+import woowacourse.omok.R
+import woowacourse.omok.database.DBController
+
+class InitController(private val omokBoard: Board, private val board: TableLayout, private val dbController: DBController) {
```

Comment:
```suggestion
class InitController(
    private val omokBoard: Board, 
    private val board: TableLayout, 
    private val dbController: DBController
) {
```
가로로 길 경우에는 개행을 해주시면 가독성이 좋아집니다.

Reply:
넵! 가독성있게 수정했습니다!